### PR TITLE
Fixes remaining UI issues

### PR DESCRIPTION
**Pull request by amitg01**
Please merge PR #58 before merging this PR.

### CODE REVIEW

**Comment by bhavesh92pandey** [comment]
@amitg01 #58 has been merged. Nico left some comments for changes in code, you can perform those in this PR #85

**Comment by amitg01** [comment]
@bhavesh I have made the changes that Nico suggested, Please have a look. If you think the changes are good enough, you can merge this PR. The only issue remaining now is the styling of the blog contents. I can raise a new PR for that.

**Comment by srijan-kapoor** [comment]
@n1c01a5 Please review and merge the changes completed in this pull request for us to move forward. @amitg01 will take care of the remaining issues in a separate PR.

File: components/blog/BlogTile.js [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-          className={styles.blogImage}
-          style={{ backgroundImage: `url(${cover})` }}
-        ></div>
         <div>
-          <div>
-            <h4 className={styles.blogTopic}>{topic}</h4>
-          </div>
-          <p className={styles.blogTitle}>{title}</p>
-          <p className={styles.blogResume}>{teaser}</p>
+          <h4 className={styles.blogTopic}>{topic}</h4>
```
Comment (n1c01a5) [inline components/blog/BlogTile.js:19]: Yes it 's better now. Just for SEO it's also better to include also a `a` link inside.

File: components/stepper/personal-details.js [hunk 2]
```diff
@@ -75,198 +80,188 @@ export default function PersonalDetails ({ handleNextStep }) {
...
+          <div className="row form-group" style={{ padding: '.375rem .75rem' }}>
             <h4>
               <span style={{ color: '#13a2dc' }}>Personal</span> Details
             </h4>
           </div>
 
-          <div className='form-group'>
-            <div className='mb-3'>
-              <label className='pdetails'>Recipient's name</label>
+          <div className="form-group">
```
Comment (n1c01a5) [inline components/stepper/personal-details.js:89]: I am not really big fan to use bootstrap for this. We don't need bootstrap it's over-engineering for this. Just `flex`was fine. But ok.

**Review by n1c01a5 — APPROVED** [review]

**Pull request merged**